js/terms.json

```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Connected organization",
+    "definition": "An organization that uses its treasury funds to establish, administer or solicit contributions to a separate segregated fund.  11 CFR 100.6(a)."
+  },
+  {
+    "term": "Coordinated",
+    "definition": "Made in cooperation, consultation or concert with, or at the request or suggestion of, a candidate, a candidate’s authorized committee or their agents, or a political party committee or its agents. 11 CFR 109.20(a)."
+  },
+  {
+    "term": "Coordinated communication",
+    "definition": "A communication that satisfies a three-pronged test: <ul class='list--numbered'><li>The communication must be paid for by a person other than a federal candidate, authorized committee, or a political party committee, or any agents of the aforementioned entities with whom the communication is coordinated.</li><li>One or more of the five content standards set forth in 11 CFR 109.21(c) must be satisfied; and</li><li>One or more of the five conduct standards set forth in 11 CFR 109.21(d) must be satisfied.</li></ul>A payment for a communication satisfying all three prongs is an in-kind contribution to the candidate or political party committee with which it was coordinated. 11 CFR 109.21."
+  },
+  {
+    "term": "Coordinated party expenditure",
+    "definition": "A special type of expenditure that can be made only by a national or state political party committee in connection with the general election of a candidate. These expenditures are subject to a separate set of limits and do not count against the party’s normal contribution limits with respect to each candidate. 11 CFR 109.30 and 109.32-37."
+  },
+  {
+    "term": "Date made",
+    "definition": "The date the contributor relinquishes control over a contribution. A contribution that is mailed is considered made on the date of the postmark. In the case of an in-kind contribution, a contribution is made on the date the goods or services are provided by the contributor. This date determines the election or calendar year limit against which a contribution counts. 11 CFR 110.1(b)(6)."
+  },
+  {
+    "term": "Date received",
+    "definition": "The date a committee (or a person acting on the committee’s behalf) takes possession of the contribution. 11 CFR 102.8(a). This date is used for FEC reporting."
+  },
+  {
+    "term": "Delegate",
+    "definition": "An individual who is or seeks to become a delegate to a national nominating convention or to a state, district or local convention, caucus or primary held to select delegates to a national nominating convention. 11 CFR 110.14(b)(1)."
+  },
+  {
+    "term": "Delegate committee",
+    "definition": "A group organized for the purpose of influencing the selection of one or more delegates. The term includes a group of delegates, a group of individuals seeking to become delegates and a group of individuals supporting delegates. 11 CFR 110.14(b)(2)."
+  },
+  {
+    "term": "Direct",
```

Comment:
Should this term be "to direct" ? even if we attach it to `direct` in context when it's used?